desktop_version/src/main.cpp

```diff
@@ -316,7 +324,16 @@ static enum LoopCode loop_run_active_funcs(void)
 
         if (index_code == Index_end)
         {
-            return Loop_continue;
+            // Loop_continue if a Func_delta exists, otherwise Loop_stop
+            for (int i = 0; i < *num_active_funcs; i++)
+            {
+                if ((*active_funcs)[i].type == Func_delta)
+                {
+                    return Loop_continue;
+                }
+            }
+
+            break;
```

Comment:
This is okay for now, but I have a better fix for the signal interrupt being ignored during unfocus pause.

Reply:
Pretty sure that fix has made it in, so we don't need this anymore.